plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -52,6 +52,14 @@ var (
 			MaxAge:    time.Hour,
 		},
 	)
+	FailedBindingLatency = prometheus.NewSummary(
```

Comment:
I think a failedBinding counter is good enough. But a latency summary also wont hurt.